src/components/ProblemsTable.tsx

```diff
@@ -369,6 +374,41 @@ export const ProblemsTable = ({
             );
           },
         },
+        displayDetailsColumn
```

Comment:
This column's visibility should be hidden using the initialState.hiddenColumns property. Here is an example: https://github.com/TanStack/table/issues/1804#issuecomment-568805891

Reply:
Another option someone else suggested would be to extend/augment the column type with an isVisible property, and filter it like how you are doing it below, except based on the value of the new isVisible property. https://github.com/TanStack/table/issues/1804#issuecomment-1547841355

Reply:
Used hiddenColumns